rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -136,7 +136,7 @@ namespace
 		case CELL_GCM_TEXTURE_A1R5G5B5:
 		case CELL_GCM_TEXTURE_R5G6B5:
 		case CELL_GCM_TEXTURE_A8R8G8B8:
-		case CELL_GCM_TEXTURE_R6G5B5:
+		case CELL_GCM_TEXTURE_R6G5B5: return { GL_ALPHA, GL_GREEN, GL_RED, GL_BLUE };
```

Comment:
I remember it works okay in 0.0.0.6 but looks like it got access violation .Probably blackout this change this moment.

{PPU[0x37] Thread (EE RenderLoop) [0x004c7a44]} struct vm::access_violation thrown: Access violation reading address 0x10